enterprise/frontend/src/metabase-enterprise/api/content-translation.ts

```diff
@@ -0,0 +1,54 @@
+import { invalidateTags, listTag } from "metabase/api/tags";
+import type { DictionaryArray } from "metabase/i18n/types";
+
+import { EnterpriseApi } from "./api";
+
+type ListContentTranslationsRequest = {
+  locale: string;
+};
+
+type UploadContentTranslationDictionaryRequest = {
+  file: File;
+};
+
+export const contentTranslationApi = EnterpriseApi.injectEndpoints({
+  endpoints: (builder) => {
+    return {
+      listContentTranslations: builder.query<
+        { data: DictionaryArray },
+        ListContentTranslationsRequest | void
+      >({
+        query: (params) => ({
+          method: "GET",
+          url: "/api/ee/content-translation/dictionary",
+          params,
+        }),
+        providesTags: () => [listTag("content-translation")],
+      }),
+      uploadContentTranslationDictionary: builder.mutation<
+        DictionaryArray,
```

Comment:
I'm not seeing this response type on upload.
![Screenshot 2025-05-02 at 8 20 50 AM](https://github.com/user-attachments/assets/6bd391cc-6f70-4478-8c4a-c66a686523fd)

Reply:
Thank you, that's a good catch